shared/attributes.asciidoc

```diff
@@ -318,6 +318,12 @@ Common words and phrases
 :multi-arg:     †footnoteref:[multi-arg,This parameter accepts multiple arguments.]
 :multi-arg-ref: †footnoteref:[multi-arg]
 
+//////////
+Reusable images
+//////////
+:y:          image:images/icon-yes.png[Yes,20,15]
```

Comment:
Would it be possible to make these `:yes-icon:` and `:no-icon:`? Just a bit easier to find across the docs than single-letter variables.

Reply:
Probably, yes. I'll make that change as well. Good suggestion 👍